content/en/docs/concepts/security/security-checklist.md

```diff
@@ -206,6 +206,15 @@ SELinux is only available on Linux nodes, and enabled in
 [some Linux distributions](https://en.wikipedia.org/wiki/Security-Enhanced_Linux#Implementations).
 {{< /note >}}
 
+## Logs and auditing
+
+- [ ] Audit logs, if enabled, are protected from general access
+- [ ] The `/logs` API is disabled (you are running kube-apiserver with
+  `--enable-logs-handler=false` and not the default setting where it is enabled).
```

Comment:
I'm not sure I agree this is necessary to disable unconditionally. If the API server is running in a container such that the only log available is its own, allowing access to it via this API could be reasonable (and may actually be the most scoped way to make that available)

Reply:
I'm in favor of defaulting the endpoint off (as https://github.com/kubernetes/kubernetes/pull/110738 does), but I wouldn't take issue with someone re-enabling it to expose only the apiserver log in a containerized environment

Reply:
The default is for the `/logs` part of the API to be enabled.

We (Kubernetes project) don't mind if people leave it enabled; the checklist is a set of sensible basic ideas, and not a formal standard. The idea is that someone who follows this checklist ends up with a reasonably well secured cluster. 

As it's a deprecated endpoint, asking people to turn it off feels reasonable. We still support its use but don't recommend that people do: [deprecation](https://dictionary.cambridge.org/dictionary/english/deprecation) means asking people to stop doing it.

Given the starting text
> This checklist aims at providing a basic list of guidance with links to more comprehensive documentation on each topic. It does not claim to be exhaustive and is meant to evolve.

then this might need to wait on https://github.com/kubernetes/website/issues/38870 so that we have something to link to.

Reply:
What advice should we state for the security checklist?